src/lib.rs

```diff
@@ -292,10 +292,6 @@ impl GlobError {
 }
 
 impl Error for GlobError {
-    fn description(&self) -> &str {
-        self.error.description()
-    }
```

Comment:
Removing functions like this is a breaking change, since callers may be using this routine to get a short description of the error. Could you please us `#[allow(deprecated)]` instead?